router/batchrouter/batchrouter.go

```diff
@@ -319,7 +328,7 @@ func (brt *HandleT) initWorkers() {
 							destUploadStat.Start()
 							output := brt.copyJobsToStorage(warehouseutils.ObjectStorageMap[brt.destType], batchJobs, true, true)
 							if output.Error == nil && output.Key != "" {
-								brt.updateWarehouseMetadata(batchJobs, output.Key)
+								output.Error = brt.postToWarehouse(batchJobs, output.Key)
```

Comment:
If copyJobsToStorage succeeded and postToWarehouse failed, we will have a dangling file in local storage and S3. Not needed right now, but we will have to fix it.

Reply:
we do a misc.RemoveFilePaths in either case